cypress/utils/utils.ts

```diff
@@ -1915,3 +1915,23 @@ export function getUniqueElementsFromSecondArray<T extends { name: string }>(
     });
     return result;
 }
+
+export function restoreColumnsToDefault(): void {
```

Comment:
These methods are not common to any other page I believe so they can be part of application model. No need to be in utils .

Reply:
As i am not initializing application in the test, should i add this method as static in application model or leave it at utils?

Reply:
Just thought in future the manage columns may be added for archetypes for example

Reply:
Hmm okay yea makes sense to leave it here then so as not to initialize applications .